ffac-autoupdater-wifi-fallback/luasrc/lib/gluon/upgrade/510-autoupdater-wifi-fallback

```diff
@@ -14,7 +14,6 @@ uci:section('autoupdater-wifi-fallback', 'autoupdater-wifi-fallback', 'settings'
   enabled = enabled,
 })
 
-uci:delete('wireless', 'fallback')
```

Comment:
wouldn't it be better to keep that for a while? it's only a cleanup, so it doesn't hurt, right?

Reply:
the former configuration did never do a `commit` on the wireless changes, so there never was something set for `wireless.fallback` in uci.
uci:save does only save the current lua changes to uci (but does not commit them to file).
This section would only be reasonable to revert changes when the device was rebooted during a autoupdater-wifi-fallback, but the uci changes are not persistet anyway, so there is nothing to clean up.

So this section never had any value, unless someone did manually set such settings.